assets/js/actions/apollo.js

```diff
@@ -102,27 +103,20 @@ export const setupApolloClient = (getAuthToken, organizationId) => {
     socket.connect()
 
     store.subscribe(async () => {
-      const newTokenClaims = await getAuthToken();
-      const newToken = newTokenClaims.__raw;
-      const newOrganization = store.getState().organization;
-
-      if (newToken !== token || currentOrganizationId !== newOrganization.currentOrganizationId) {
```

Comment:
none of this ever does anything, switch orgs reloads the page entirely, bypassing this